src/script/ceph-backport.sh

```diff
@@ -1400,6 +1400,16 @@ else
     abort_due_to_setup_problem
 fi
 
+#
+# do we have jq available?
+#
+
+if type jq >/dev/null 2>&1 ; then
```

Comment:
```suggestion
if which jq; then
```

Reply:
We have replaced `which` with `type` (or another shell built-in whose name I seem to have forgotten) in a number of script already, because they break in environments that don't have `which` installed. So I'm against using `which` here.

Reply:
Wow, `type` is more available than `which`? That's very surprising...

Reply:
@batrick This script is supposed to be run with `bash` (see the shebang at the top). And with bash:

```
$ type type
type is a shell builtin
```

So, yes, `type` is "guaranteed" (by the fact that the script is supposed to be running in a bash shell process) to be available. The same cannot be said for `which`.

Reply:
(TBH I was surprised to learn this, too...)